test/tests/varnish/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eo pipefail
+
+set -x
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+vname="varnish-volume-$RANDOM-$RANDOM"
+trap "docker volume rm $vname > /dev/null" EXIT
+docker volume create --driver local \
+	--opt type=tmpfs \
+	--opt device=tmpfs \
+	--opt o=size=100m \
+	$vname
+
+
+cname="varnish-container-$RANDOM-$RANDOM"
+cid="$(
+	docker run -d \
+		--network host \
+		-v $vname:/var/lib/varnish \
+		--name "$cname" \
+		"$image" \
+		varnishd -F -a 0:0 -f /etc/varnish/default.vcl
+)"
+trap "docker rm -vf $cid > /dev/null; docker volume rm $vname > /dev/null" EXIT
+
+sidecar() {
+	docker run --rm -i \
+		--network host \
```

Comment:
To keep things isolated and ensure we don't ever run into port conflicts, we should drop `host` networking and just use `--network container:"$cid"` for the sidecar.

Reply:
nice, I wasn't aware that it was an option! Fixed